app/controllers/applications_controller.rb

```diff
@@ -15,14 +15,14 @@ def create
       redirect_to "/applications/#{application.id}"
     else
       flash[:notice] = "Please fill in answers to all prompts"
-      redirect_to "/applications/new"
+      render :new
     end
```

Comment:
changed this after I asked Kat the difference and she said doing a render :new is more efficient (one less transaction cost) than doing a redirect

Reply:
Nice, I was thinking of finding a way to work a render in too. Thanks for doing this